CODEOWNERS

```diff
@@ -18,6 +18,7 @@
 /stable/ecr-cleanup/ @sudermanjr
 /stable/rbac-manager/ @robscott @sudermanjr @lucasreed
 /stable/polaris/ @rbren @makoscafee @jordandoig @baderbuddy
+/stable/gemini/ @rbren
```

Comment:
perhaps @dosullivan would like to join you? I think it would be good to have more than one for a stable chart.